mobile.html

```diff
@@ -20,14 +20,141 @@ <h1 class="mylogo">MyLogo</h1>
             <li class="connect">LET’S CONNECT</li>
         </ul>
         <section class="icons">
-            <img src="./images/icons/git.png" alt="git">
             <img src="./images/icons/twitter.png" alt="twitter">
             <img src="./images/icons/Linkedin.png" alt="linkedin">
             <img src="./images/icons/M.png" alt="m">
+            <img src="./images/icons/git.png" alt="git">
             <img src="./images/icons/2.png" alt="2">
         </section>
     </section>
 
-    
+    <section class="card1">
```

Comment:
All the cards we need to wrap in just 1 section. 
You can have something similar to this:

< section class="projects" > 
.....< div class="card" >...< /div >
.....< div class="card" >...< /div >
  .
  .
  .
< /section >

The section needs to have the style set to grid.
It is not needed and is not recommended to have other `section` tags inside of a `section` tag.
A lot of your CSS code repeats itself because you have different classes for cards. It is enough if you give the same class to all cards.

- [ ] Kindly, implement the above requirements

Reply:
hi nicolla, thanks a lot this one-class idea for all cards is great, it minimized my CSS code.
now I have made all the required changes you asked for, but as I mentioned before I face a problem with the height of cards, any suggestion?

Reply:
or should I give them all the greatest card height?

Reply:
@sameh080081  You can do that too, as long as it keeps the figma design. But if you want, you can add another classes for the attributes that are not repeting